engine/runtime/src/test/scala/org/enso/compiler/test/pass/resolve/TypeSignaturesTest.scala

```diff
@@ -252,10 +252,12 @@ class TypeSignaturesTest extends CompilerTest {
 
     val ir =
       """
-        |f a (b = 1 : Int) : Double
+        |f a (b = 1 : Int) = 42.5
         |""".stripMargin.preprocessExpression.get.resolve
 
     // FIXME: Not supported by new parser--needs triage (#5894).
+    // Original: `f a (b = 1 : Int) : Double` makes no sense, right Kaz?
+    // anyway I don't see any `TypeSignatures` anywhere even after changing the expression
```

Comment:
I think the original is testing what we now call type-annotated expressions, and that seems like a better parse for this case than the current `TypeSignature`. Created #6152.

Reply:
Added a link in d880d7e278